<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsWFSSourceSelectBase</class>
 <widget class="QDialog" name="QgsWFSSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add WFS Layer from a Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QCheckBox" name="mHoldDialogOpen">
     <property name="text">
      <string>Keep dialog open</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QgsFilterLineEdit" name="lineFilter">
     <property name="showSearchIcon" stdset="0">
      <bool>true</bool>
     </property>
     <property name="qgisRelation" stdset="0">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="cbxFeatureCurrentViewExtent">
     <property name="text">
      <string>Only request features overlapping the view extent</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Server Connections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbConnections"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Connect to selected service</string>
          </property>
          <property name="text">
           <string>C&amp;onnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNew">
          <property name="toolTip">
           <string>Create a new service connection</string>
          </property>
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Edit selected service connection</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Remove connection to selected service</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>171</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="toolTip">
           <string>Load connections from file</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="toolTip">
           <string>Save connections to file</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutFilter"/>
   </item>
   <item row="9" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Help</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QGroupBox" name="gbCRS">
     <property name="title">
      <string>Coordinate Reference System</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCoordRefSys">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>441</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnChangeSpatialRefSys">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Changeâ€¦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>cbxFeatureCurrentViewExtent</tabstop>
  <tabstop>btnChangeSpatialRefSys</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
